### PR TITLE
Enable PR review requirement

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

**Comment by cevich** [comment]
@edsantiago if you don't mind.  Please review and mark 'approved'.  I want to test/make sure the automatic merge system works.

File: .mergify.yml [hunk 1]
```diff
@@ -15,8 +15,7 @@ pull_request_rules:
           - '-title~=.*CI.*SKIP'  # Cirrus-CI feature
           - '-title~=.*SKIP.*CI'  # "
           - 'status-success=success'  # defined in .cirrus.yml
-          # Temporarily disabled during repository bootstrapping
-          # - '#approved-reviews-by>=1'
+          - '#approved-reviews-by>=1'
```
Comment (edsantiago) [inline .mergify.yml:18]: I have to assume that the `#` here, inside a string, is an actual part of the conditional. Is this under your control? Is there any possible way to remove the hash sign? It's super confusing. But I'll approve anyway.

Comment (cevich) [reply .mergify.yml:18]: confirmed, here's the spot in docs: https://doc.mergify.io/conditions.html#conditions

**Comment by cevich** [comment]
Ahh, this one isn't going to auto-merge because the `.mergify.yml` file is modified...one of the guard-rails it has in place.  No worries.

**Comment by cevich** [comment]
Thanks @edsantiago

**Pull request merged**